desktop/cmp/contextmenu/ContextMenuSupport.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {defaultMethods} from '@xh/hoist/utils/ClassUtils';
+import {contextMenu} from './ContextMenu';
+import {ContextMenuTarget} from '@xh/hoist/kit/blueprint';
+
+/**
+ * Mixin to allow a Component to show a ContextMenu.
+ *
+ * Components that use this mixin should implement a getContextMenuItems()
+ * method.
+ *
+ * See Blueprint documentation for more information about the implementation of this
+ * behavior.
```

Comment:
I would tuck the orphaned words up with their respective lines above. Only mention as you were inquiring about my CLBOCD (comment-line-break-OCD) the other day 😀